app/model/Payment/Payment.php

```diff
@@ -343,17 +343,20 @@ private function checkNotClosed() : void
         }
     }
 
+    /**
+     * @param string[] $recipients
+     */
     private function updateDetails(
         string $name,
-        ?string $email,
+        array $recipients,
         Date $dueDate,
         ?int $constantSymbol,
         string $note
     ) : void {
-        $this->name           = $name;
-        $this->email          = $email;
-        $this->dueDate        = $dueDate;
-        $this->constantSymbol = $constantSymbol;
-        $this->note           = $note;
+        $this->name            = $name;
+        $this->dueDate         = $dueDate;
+        $this->constantSymbol  = $constantSymbol;
+        $this->note            = $note;
+        $this->emailRecipients = new ArrayCollection(array_map(fn(string $recipient) => new EmailRecipient($this, $recipient), $recipients));
```

Comment:
Tady by mělo být buď:
a) validace, že tam nejsou duplicitní maily
b) vyfiltrování duplicitních mailů

osobně mi přijde přívětivější b), každopádně jedno z toho by se zde mělo dít a mělo by to být otestováno.

Reply:
doplněno array_unique a otestováno